make -C tests
make[1]: Entering directory '/home/user/tmp/rset-3.0/tests'
ld -r ../compat.o ../rutils.o ../input.o ../execute.o -o rset.o -Wl,--as-needed,-O1,--sort-common
ld: unrecognized option '-Wl,--as-needed,-O1,--sort-common'
ld: use the --help option for usage information
make[1]: *** [Makefile.bsd:18: rset.o] Error 1
make[1]: Leaving directory '/home/user/tmp/rset-3.0/tests'
make: *** [Makefile.bsd:78: test] Error 2

--- a/tests/Makefile.bsd
+++ b/tests/Makefile.bsd
@@ -17,7 +17,7 @@ RSET_LIBS = ../compat.o ../rutils.o ../i
 all: rset.o test
 
 rset.o: ${RSET_LIBS}
-	${LD} -r ${RSET_LIBS} -o $@ ${LDFLAGS}
+	${LD} -r ${RSET_LIBS} -o $@
 
 .c: rset.o ${RSET_LIBS}
 	${CC} -I.. ${CFLAGS} ${CPPFLAGS} $< rset.o -o $@ ${LDFLAGS}
